page/upgrade-guide/1.12.md

```diff
@@ -516,3 +516,26 @@ $( ".from" ).transfer( {
 	to: $( ".target" )
 } );
 ```
+
+### Added `.cssClip()`
+
+The new [`.cssClip()` method](//api.jqueryui.com/cssClip/) is a convenience method for getting and setting the CSS `clip` property as an object. The object contains `top`, `right`, `bottom`, and `left` properties. For example, if you want to clip off 10 pixels from the right side of an element:
+
+```js
+var element = $( ".elem" );
+var clip = element.cssClip();
+clip.right -= 10;
+element.cssClip( clip );
+```
+
+### Added low-level effects APIs
+
+A number of new low-level effects APIs have been created to aid in the development of custom effects.
+
+* [`$.effects.define()`](//api.jqueryui.com/jQuery.effects.define/): Defines an effect.
+* [`$.effects.scaledDimensions()`](//api.jqueryui.com/jQuery.effects.scaledDimensions/): Scales the dimensions of an element.
+* [`$.effects.clipToBox()`](//api.jqueryui.com/jQuery.effects.clipToBox/): Calculates position and dimensions based on a clip animation.
+* [`$.effects.saveStyle()`](//api.jqueryui.com/jQuery.effects.saveStyle/): Saves all inline styles applied to an element.
```

Comment:
Sure. These are copied directly from the descriptions on api.jqueryui.com, so I'll update it there first.